cypress/integration/core_opensearch_dashboards/opensearch_dashboards/apps/query_enhancements/04/s3_dataset.spec.js

```diff
@@ -100,6 +100,10 @@ const s3DatasetTestSuite = () => {
         });
         afterEach(() => {
           cy.deleteWorkspaceByName(workspace);
+          cy.window().then((win) => {
+            win.localStorage.clear();
```

Comment:
is `cy.clearLocalStorage()` and `cy.clearAllSessionStorage()`
source: https://docs.cypress.io/api/commands/clearlocalstorage

not available?

also should we consider the clearing out in the `beforeEach`? in case a different test file doesn't follow best practices and clears out this info. so ensuring that this test run is clean. regardless if the previous test cleared out local storage and session storage or not